### PR TITLE
Added modelHome variable for Verrazzano/WKO templates

### PR DESCRIPTION
**Pull request by ddsharpe**
Provide documentation for --resourceTemplates parameter, and added {{modelHome}} as a new token for resource templates.

### CODE REVIEW

**Review by jshum2479 — APPROVED** [review]

File: site/create-image.md [hunk 1]
```diff
@@ -99,6 +100,19 @@ installation or domain creation steps, use the `final-build-commands` section so
 final stage of the image build.  Or, if the file needs to change the Oracle Home prior to domain creation, use 
 the `after-fmw-install` or `before-wdt-command` sections.
 
+#### Resource Template Files
+
+If provided, the file or files provided with `--resourceTemplates` will be overwritten. For known tokens, 
+the placeholders will be replaced with values according to the table below.  **Note:** Placeholders must follow
```
Comment (rosemarymarano) [inline site/create-image.md:106]: according to the table below -> according to the following table

File: site/update-image.md [hunk 2]
```diff
@@ -109,6 +110,19 @@ The `latestPSU` option will continue to be supported for the CREATE option, but
 UPDATE option.  Because of the number of patches and their size, using `latestPSU` as an update to an existing image can 
 increase the size of the image significantly, and is not recommended. 
 
+#### Resource Template Files
+
+If provided, the file or files provided with `--resourceTemplates` will be overwritten. For known tokens, 
+the placeholders will be replaced with values according to the table below.  **Note:** Placeholders must follow
```
Comment (rosemarymarano) [inline site/update-image.md:116]: according to the table below -> according to the following table

**Review by rosemarymarano — APPROVED** [review]
minor edit

**Review by rakillen — APPROVED** [review]

File: site/update-image.md (hunk 2 above)
Comment (ddsharpe) [reply site/update-image.md]: Thank you, fixed.

File: site/create-image.md (hunk 1 above)
Comment (ddsharpe) [reply site/create-image.md]: Thank you, fixed.

**Pull request merged**